brave/brave-core
# stats: Have usage pings send API key

Resolves brave/brave-browser#9911

## Submitter Checklist:

- [b] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on
- [] Ran `git rebase master` (if needed).
- [] Ran `git rebase -i` to squash commits (if needed).

## Test Plan:

 1. Get custom build of browser with stats server set to https://laptop-updates-pre.brave.com. Required change in files `browser/brave_stats_updater.cc` and `common/network_constants.cc`
 2. Clear browser local state
 3. Launch browser and watch network requests (Wireshark, Charles)
 4. Ensure that api_key is sent to laptop-updates server

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
